std/machines/test_util.asm

```diff
@@ -0,0 +1,13 @@
+/// A too-small version of std::machines::range::Byte2.
+/// It is used in tests which happen to only use the first 256 values.
```

Comment:
Shouldn't this move to `test_data` then?

Reply:
done